src/Contracts/Audit.php

```diff
@@ -4,6 +4,14 @@
 
 interface Audit
 {
+    /**
+     * Create a new Eloquent query builder for the model.
+     *
+     * @param  \Illuminate\Database\Query\Builder  $query
+     * @return \Illuminate\Database\Eloquent\Builder<\Illuminate\Database\Eloquent\Model>|static
+     */
+    public function newEloquentBuilder($query);
```

Comment:
You can't change an interface like this, that could become a breaking change

If it's only for PhpStan try adding this at the start of the interface

```php
/**
 * @mixin \Illuminate\Database\Eloquent\Model
 */
interface Audit
{
```

Reply:
Also you could add 
```php
/*
 * @method static \Illuminate\Database\Query\Builder newEloquentBuilder(\Illuminate\Database\Query\Builder  $query)
 */
interface Audit
{
```